### PR TITLE
[8.0] constraints enable cancel

### PR DESCRIPTION
**Pull request by NL66278**
Forward port of PR for 7.0: https://github.com/OCA/account-financial-tools/pull/454

### CODE REVIEW

**Comment by elicoidal** [comment]
Travis is failing

File: account_constraints_banking/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <https://therp.nl>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Account Constraints Banking',
+    'version': '7.0.1.0.0',
```
Comment (elicoidal) [inline account_constraints_banking/__openerp__.py:6]: 8

File: account_constraints_banking/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline account_constraints_banking/__init__.py:1]: Can you add a README

File: account_constraints_banking/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    'depends': [
+        'account_constraints',
+        'account_banking',
+    ],
+    'author': "Therp BV,Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
+    'category': 'Generic Modules/Accounting',
+    'description': """
```
Comment (elicoidal) [inline account_constraints_banking/__openerp__.py:14]: move to README

File: account_constraints_banking/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+linked to a bank statement, directing the user to instead apply changes
+on the bank statement directly. Without this module changes on the
+bank statement, like cancelling a transaction, are also prevented.
+
+  Contributors
+  * Ronald Portier (Therp BV)  <ronald@therp.nl>
+
+    """,
+    'website': 'https://therp.nl',
+    'data': [],
```
Comment (elicoidal) [inline account_constraints_banking/__openerp__.py:30]: remove empty keys

File: account_constraints_banking/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+# © 2017 Therp BV <https://therp.nl>.
```
Comment (elicoidal) [inline account_constraints_banking/__init__.py:2]: Copyright date is probably incorrect?

**Review by elicoidal — CHANGES_REQUESTED** [review]
not sure about the functional usage.
Needs rework

**Comment by NL66278** [comment]
This was a forward port of function needed in 7.0 to make account_constraints work properly in combination with banking. After more carefull consideration of the existing code, the problems solved by this module have already been dealt with in the existing 8.0 version of account_constraints.

**Pull request closed without merging**